src/CSS.php

```diff
@@ -637,7 +637,7 @@ protected function stripEmptyTags($content)
      */
     protected function stripEmptyProperties($content)
     {
-        return preg_replace('/[a-z-:().#]*:\s*;/', '', $content);
+        return preg_replace('/[a-zA-Z-:().#]*:\s*;/', '', $content);
```

Comment:
Technically this should also support numbers and underscores: https://www.w3.org/TR/css-syntax-3/#ident-token-diagram